src/rastervision/rv_config.py

```diff
@@ -94,6 +97,12 @@ def __init__(self,
                  verbosity=Verbosity.NORMAL):
         self.verbosity = verbosity
 
+        # Set logging level
+        log = logging.getLogger('rastervision')
+        if self.verbosity >= Verbosity.NORMAL:
+            log.setLevel(logging.INFO)
+        log = logging.getLogger(__name__)
```

Comment:
Is `log` unused here? If this is to avoid setting a logger that is based on `rastervision` since it's used above, I'd recommend giving that variable another name (e.g. `root_log = logging.getLogger('rastervision')`)